generic-worker.Dockerfile

```diff
@@ -53,7 +53,7 @@ else \
     echo \{\"version\": \"69.0.1\", \"commit\": \"local\", \"source\": \"https://github.com/taskcluster/taskcluster\", \"build\": \"NONE\"\} > /version.json; \
 fi
 
-VOLUME /etc/generic-worker/config.json
```

Comment:
I'm not sure how this worked before, aiui VOLUME should be a directory?

Reply:
good catch!